### ISSUE TITLE
Cannot read property 'position' of undefined

### ISSUE THREAD

**Issue by r0b0t3d** [Esri/react-sceneview#49]
Hi,
I am trying with this library and got this error.
It come from `CameraEventListener `
```
listeners.push(this.props.view.watch('stationary', () => this.handleUpdate()));
```
that trigger
```
handleUpdate() {
    this.props.onCameraChange({
      camera: {
        position: this.props.view.camera.position,
        heading: this.props.view.camera.heading,
        tilt: this.props.view.camera.tilt,
        scale: this.props.view.scale,
      },
      stationary: this.props.view.stationary,
      interacting: this.props.view.interacting,
    });
  }
```
And `this.props.view.camera` is `undefined`

Here is my component
```
return (
      <SceneView id="sceneview">
        <Scene />
      </SceneView>
    )
```

**Comment by r0b0t3d** [comment]
Above error is gone when adding `goTo` to `SceneView`. But I got another issue
```
react-sceneview.js:18752 Uncaught (in promise) TypeError: Cannot read property 'interacting' of null
    at SceneView._callee2$ (react-sceneview.js:18752)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:264)
    at Generator.prototype.<computed> [as next] (runtime.js:98)
    at src_asyncGeneratorStep (react-sceneview.js:18586)
    at _next (react-sceneview.js:18608)
```

**Comment by sandromartis** [comment]
The first issue has been fixed in v0.2.4

Not sure about the second issue. I'm going to close this. Please let us know if the issue still persists.
